client/src/components/VolunteersForm.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+
+
+class VolunteersForm extends React.Component {
+state={
+    firstName:"",
+    lastName:"",
+    email:"",
+    phoneNumber:"",
+    password:"",
+    address:""
+
+};
+onSubmit=e=>{
+    e.preventDefault();
+    console.log(this.state);
```

Comment:
It's great that you added this until we can think of how we're going to send emails. It keeps this pull request small and easier to review! 👍

Reply:
ok .I do my all